ui/app/models/sync/destination.js

```diff
@@ -20,6 +20,9 @@ const validations = {
 export default class SyncDestinationModel extends Model {
   @attr('string', { subText: 'Specifies the name for this destination.', editDisabled: true }) name;
   @attr type;
+  // only present if delete action has been initiated
+  @attr('string') purgeInitiatedAt;
+  @attr('string') purgeError;
```

Comment:
I think the serializer should also be updated to remove these properties from the payload.

Reply:
Sure! It'd be pretty odd for a user to edit a destination that failed deletion but I can remove them from the payload!

Reply:
Ah yeah - they didn't show up when I was testing. But doesn't hurt to account for just in case